app/lib/features/chat_ng/providers/chat_room_messages_provider.dart

```diff
@@ -10,6 +10,32 @@ import 'package:logging/logging.dart';
 import 'package:riverpod/riverpod.dart';
 
 final _log = Logger('a3::chat::message_provider');
+const _supportedTypes = [
+  'm.room.member',
+  'm.room.message',
+  'm.room.redaction',
+  'm.room.encrypted',
+  'm.policy.rule.room',
+  'm.policy.rule.server',
+  'm.policy.rule.user',
+  'm.room.aliases',
+  'm.room.avatar',
+  'm.room.canonical_alias',
+  'm.room.create',
+  'm.room.encryption',
+  'm.room.guest.access',
+  'm.room.history_visibility',
+  'm.room.join.rules',
+  'm.room.name',
+  'm.room.pinned_events',
+  'm.room.power_levels',
+  'm.room.server_acl',
+  'm.room.third_party_invite',
+  'm.room.tombstone',
+  'm.room.topic',
+  'm.space.child',
+  'm.space.parent',
```

Comment:
I'd rather we don't add them here until we have a proper support. Like showing a "has changed the topic to XYZ" ... Let's not do the same mistake of just doing brush-of-a-stroke not-really-thought-through rendering please ... rather keep them off screen until we have decent support...